tests/components/generic/test_camera.py

```diff
@@ -275,7 +275,9 @@ async def test_limit_refetch(
 
     with (
         pytest.raises(aiohttp.ServerTimeoutError),
-        patch("asyncio.timeout", side_effect=TimeoutError()),
+        patch.object(
+            client.session._connector, "connect", side_effect=asyncio.TimeoutError
+        ),
```

Comment:
No timeout is started on connection reuse anymore since there is nothing to wait for